globals.go

```diff
@@ -192,6 +192,7 @@ const (
 	ExpireOnFormat                      = "2006-01-02"
 	DefaultMaxSpawnHostsPerUser         = 3
 	DefaultSpawnHostExpiration          = 24 * time.Hour
+	DefaultSecurityGroup                = "sg-097bff6dd0d1d31d0"
```

Comment:
Could you add this as a setting on the admin page? We try to avoid using constants for parts of our infrastructure, since these can change, and in order to keep Evergreen possible to run outside of our specific setup.